jerry-core/ecma/base/ecma-helpers.c

```diff
@@ -797,7 +797,7 @@ ecma_free_property (ecma_object_t *object_p, /**< object the property belongs to
     default:
     {
       JERRY_UNREACHABLE ();
-      return;
+      break;
```

Comment:
Why do you change this?

Reply:
Same as above. Don't try to do anything meaningful after a failed assert. Style guide seems to expect a `break` at the end of every switch case, even in `default` (at least that's how most of the code base looks like), so be it. But after the unreachable point, nothing should happen.

Reply:
Merge it please just like above.